.github/vale/styles/Klaw/branding.yml

```diff
@@ -22,3 +22,5 @@ swap:
   "node[ -]?js?": NodeJS
   "type[ -]?scripts?": TypeScript
   "(?i)ssl": SSL
+  "(?i)klaw": Klaw
```

Comment:
Hi! Could you remove that rule and make sure it is still in `branding-warning-temp.yml`? This should (for now) only be an error, because it's not fixed yet.